src/app/pcasts/controllers/get_new_episode_notification_controller.py

```diff
@@ -0,0 +1,19 @@
+from . import *
+
+class GetNewEpisodeNotificationController(AppDevController):
+
+  def get_path(self):
+    return '/notifications/episodes/'
+
+  def get_methods(self):
+    return ['GET']
+
+  @authorize
+  def content(self, **kwargs):
+    user = kwargs.get('user')
+    max_episodes = request.args['max']
```

Comment:
por que no casting max?

Reply:
both offset and max are only used by the SQLAlchemy which does not require casting

Reply:
Will be removed in feedback